packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -92,12 +93,13 @@ const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>(
         paddingX={paddingX}
         paddingY={paddingY}
         verticalAlign={verticalAlign}
+        i18nExternalLinkLabel={i18nExternalLinkLabel}
```

Comment:
Suggestion: it looks like you don't need to pass the i18n prop to the variant, since it is only used on line 102.